### ISSUE TITLE
Create unit test cases

### ISSUE THREAD

**Issue by aayushai** [aayushai/my-list-service#3]
I have made the test cases but some test case are getting failed, if the issue is with the code kindly check it or update the test cases.

**Comment by SureshPradhana** [comment]
@aayushai I would like to work on this?

**Comment by aayushai** [comment]
Sure Go Ahead! @SureshPradhana

**Comment by SureshPradhana** [comment]
![image](https://github.com/user-attachments/assets/14854edc-40dd-4215-83f6-446698d2ee70)

Only a single test case is running.

**Comment by aayushai** [comment]
Yes I know that's why i raised issue. I need help with the code.
